servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/ClientValidationTests.java

```diff
@@ -59,7 +59,10 @@ public static void createEntities() throws ExecutionException, InterruptedExcept
 
 	@AfterClass
 	public static void deleteEntities() throws ExecutionException, InterruptedException, IOException {
-		managementClient.deleteQueueAsync(queuePath).get();
+		if (managementClient == null) {
+		    return;
+        }
```

Comment:
It looks like this file has tabs in it, rather than spaces, hence the odd indentation here. Please resolve this so that the code is indented correctly.

Reply:
updated